### PR TITLE
AF-2125: Remove unused dependencies

### PR DESCRIPTION
**Pull request by jomarko**
AF-2125 caused some compilation issues, see:
- https://github.com/kiegroup/kie-wb-common/commit/a843ccffd7f3dd82c80cd72d25ea1c1c6c91c67f
- https://github.com/kiegroup/kie-wb-common/pull/2977

During investigation also unused dependencies were found. This PR removes them.

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins please retest this
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by romartin** [comment]
ok thanks @jomarko , just asking for a quick QE review, all green, so once having the approval will merge. Thx!

File: kie-wb-common-kogito/kie-wb-common-kogito-client/pom.xml [hunk 1]
```diff
@@ -64,6 +64,10 @@
       <artifactId>uberfire-commons-editor-client</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-widgets-core-editors</artifactId>
```
Comment (jomarko) [inline kie-wb-common-kogito/kie-wb-common-kogito-client/pom.xml]: Hmmm, the mystery continues, now `mvn dependency:analyze` marks this as unused declared dependency.

**Comment by jomarko** [comment]
@romartin @hasys sorry, I double checked affected modules and seems still some updates will be needed. Please postpone merge.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
Failed tests:
- org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testLibraryPerspectiveWithHeader
- org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by hasys** [comment]
Hi @jomarko,

is mentioned tests instability reported?

> Failed tests:
> 
>     * org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.testLibraryPerspectiveWithHeader
> 
>     * org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest.org.kie.wb.selenium.ui.StandalonePerspectivesIntegrationTest

**Comment by jomarko** [comment]
@hasys I do not think so

**Comment by jomarko** [comment]
@hasys @romartin the build passed, may I ask you for a re-review?

**Pull request merged**